pyproject.toml

```diff
@@ -25,7 +25,7 @@ dependencies = [
     "macaroonbakery>=1.1,<2.0",
     "pyRFC3339>=1.0,<2.0",
     "pyyaml>=5.1.2",
-    "websockets>=8.1,<14.0",
+    "websockets>=13.0.1,<14.0",
```

Comment:
[not a blocker] Nice, I just checked and 13 is the last major version of websockets to support python 3.8, so this works for us =)

Reply:
IIRC type hints got improved in 12 and/or 13.